ci/environment.yml

```
@@ -14,6 +14,6 @@ dependencies:
   - pydap
   - pytest
   - pytest-cov
-  - shapely
+  - shapely>=2
```

```suggestion
  - shapely
```
Is there a reason for the re-pinning of shapely here ?